doc/extdev/index.rst

```diff
@@ -189,6 +189,89 @@ as metadata of the extension.  Metadata keys currently recognized are:
   usually don't negatively influence the process, this defaults to ``True``.
 
 
+Extension How-tos
+-----------------
```

Comment:
Is this change intended? It seems not related to "events overview".

Reply:
nope, accident, I'll remove